seed_data/utils.py

```diff
@@ -64,3 +65,11 @@ def create_past_date_range(ended_days_ago=5, day_spread=10, now=None):
 def future_date(now=None, days_in_future=30):
     """Creates a date in the future"""
     return now + timedelta(days=days_in_future)
+
+
+def localized_datetime(dt, tz='US/Eastern'):
```

Comment:
moment.js on the front end is parsing these dates in the local timezone (US/Eastern). this function is only being used by a convenience method that spits out course information. it's being used by jared right now to aid in testing our various course states. this allows him to match up the dates that are being shown to him on the front end with the actual data, and that matching can only happen when the dates are localized consistently